apps/site/src/components/pages/pricing/paid-addons-section.tsx

```diff
@@ -0,0 +1,658 @@
+                >
+                  Find out more
+                  <FontAwesomeIcon
+                    icon={faChevronRight}
+                    sx={{ fontSize: 14 }}
+                  />
+                </CustomLink>
+              </Typography>
+            </Stack>
+          </Section>
+
+          <Section sectionName="dev">
+            <Stack flexDirection="row" alignItems="center" mb={2.25}>
+              <GradientFontAwesomeIcon
+                icon={faHammer}
+                sx={{ mr: 1.5, fontSize: 32 }}
+                light
+              />
+
+              <Typography
+                variant="bpHeading3"
+                sx={{
+                  lineHeight: 1.4,
+                  color: ({ palette }) => palette.gray[90],
+                }}
+              >
+                Dev Services
+              </Typography>
+            </Stack>
+
+            <Stack gap={2.25}>
+              <Typography variant="bpBodyCopy" sx={{ lineHeight: 1.4 }}>
+                <strong>
+                  Anybody can suggest amendments to the Þ specification.
+                </strong>{" "}
+                The protocol is openly debated and governed, and you don’t need
+                to hire us (in fact, you can’t!) in order to change the spec.
+              </Typography>
+
+              <Typography variant="bpBodyCopy" sx={{ lineHeight: 1.4 }}>
+                <strong>Similarly, anyone can build a Þ block.</strong> However,
+                if you’d like a block built for you, or are looking to partner
+                on a particularly complex block, we provide support or can
+                connect you with a community developer.
+              </Typography>
+
+              <Box mb={1.5}>
+                <CustomLinkButton
+                  href="/contact"
+                  endIcon={<FontAwesomeIcon icon={faEnvelope} />}
+                >
+                  Contact us
+                </CustomLinkButton>
+              </Box>
+            </Stack>
+          </Section>
+
+          <Section sectionName="support">
+            <Stack flexDirection="row" alignItems="center" mb={2.25}>
+              <GradientFontAwesomeIcon
+                icon={faSlack}
+                sx={{ mr: 1.5, fontSize: 32 }}
+                light
+              />
+
+              <Typography
+                variant="bpHeading3"
+                sx={{
+                  lineHeight: 1.4,
+                  color: ({ palette }) => palette.gray[90],
+                }}
+              >
+                1:1 Support
+              </Typography>
+            </Stack>
+
+            <Stack gap={2.25}>
+              <Typography variant="bpBodyCopy" sx={{ lineHeight: 1.4 }}>
+                <strong>
+                  We offer dedicated 1:1 premium support with guaranteed SLAs
+                  via email and Slack.
+                </strong>
+              </Typography>
+
+              <List sx={{ marginLeft: 3, listStyle: "disc" }}>
```

Comment:
Can we add a bit of margin below each bullet? They're a bit squashed together – I think the gap between lines within a bullet is actually greater than the gap between bullets

Reply:
Added `12px` of spacing between bullet items in 73ea8736a11c5c2b26aa74acc3e7b367bca76de8